app/konnect/api-products/service-documentation.md

```diff
@@ -24,9 +18,25 @@ You can provide extended descriptions of your API products with a Markdown (`.md
   <img src="/assets/images/products/konnect/api-products/konnect_service_docs_description.png" />
 </p>
 
-All API product descriptions are managed from the **Documentation** section in the API Product overview. Once you've uploaded the markdown file, you have a preview of how it will render, the option to edit, and a view of the publication status. You can also create a hierarchy between the docs you upload that will be reflected in the way they're displayed in the Dev Portal.
 
-## API specification
+Manage all API product descriptions from the Documentation section in the API Product overview. After uploading your markdown file, you'll be able to preview its rendering, edit it directly using the {{site.konnect_saas}} Markdown Renderer, and check its publication status. Additionally, you can organize your documents into a hierarchy, which will influence the order they are displayed to your users from within the Dev Portal.
+
+### Interactive markdown renderer
```

Comment:
@Guaris I just took a look at the preview and everything looks great, IMO!

Just one small thing, which you can ignore since it's not yet pertinent -- This markdown renderer is not only being used in API Products, but also in Service Hub. When the time comes for Service Hub to go Public Beta, it might make sense to separate out a dedicated Kong Docs page that is solely for the Kong Konnect Markdown Renderer. And then, the Docs pages for API Products and Service Hub could separately just link to it.

Reply:
Also, just so you have additional context, another problem we will eventually face is that when a customer uploads a new document (and this is true in either API Products or Service Hub), there is a "View Documentation" link that takes them here: https://docs.konghq.com/konnect/api-products/service-documentation/#main

This experience is probably fine if they are in API Products, but not great if they are coming from Service Hub!
<img width="2032" alt="Screenshot 2024-03-26 at 7 54 01 PM" src="https://github.com/Kong/docs.konghq.com/assets/137116765/80ce1d23-43eb-4861-9370-ac1ac1e5a050">

Reply:
@miko-bautista 
I agree! 
I made this ticket to make those changes for service hub https://konghq.atlassian.net/browse/DOCU-3755. I saw the notes in your draft, but I thought that without having the service hub available right now it would be more confusing.